src/quick-lint-js/error.h

```diff
@@ -926,6 +926,10 @@
       WARNING(QLJS_TRANSLATABLE("redundant delete statement on variable"),     \
               delete_expression))                                              \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_missing_if_after_else, "E0184", { source_code_span expected_if; }, \
+      ERROR(QLJS_TRANSLATABLE("missing if after else"), expected_if))          \
```

Comment:
Write quotes around code:
```suggestion
      ERROR(QLJS_TRANSLATABLE("missing 'if' after 'else'"), expected_if))          \
```